tests/unit/test_schemas.py

```diff
@@ -236,23 +187,26 @@ def validateClasses(self, factory):
         for cls in protocol.getProtocolClasses():
             instance = factory(cls)
             jsonDict = instance.toJsonDict()
-            self.assertTrue(cls.validate(jsonDict))
+            if not cls.validate(jsonDict):
+                print(cls.__name__)
+                print(avrotools.ValidationTool.getInvalidFields(
+                    cls, jsonDict))
+                self.fail("validation failed")
 
-    @unittest.skipIf(protocol.version.startswith("0.6"), "")
     def testValidateDefaultValues(self):
         self.validateClasses(self.getDefaultInstance)
 
-    @unittest.skipIf(protocol.version.startswith("0.6"), "")
     def testValidateTypicalValues(self):
-        self.validateClasses(self.getTypicalInstance)
+        self.validateClasses(avrotools.CreationTool.getInstance)
 
+    # TODO fix ReadAlignment.alignments decode type ambiguity!
```

Comment:
I spent much of Friday thinking about this along with other issues having to do with mixing "classic" and graph modes into the same "do it all" entities. While I'm mulling how to make a schemas PR to unbundle such objects,  how about something like this for now:
```
union { null, LinearAlignment } alignment = null;
union {null, GraphAlignment } graphAlignment = null;
```
with the intention of only ever having one of these non-null - we're already using this kind of logic all over the place.